docs/features/medication-log-phase2.md

```
@@ -0,0 +1,173 @@
+  - 두 schedule (1정, 1정), Vision=1 → COUNT_MISMATCH
+  - dosage="반알" 포함 → COUNT_UNKNOWN, Vision 호출 안 됨 (verify mock no interaction)
+  - Vision Optional.empty() → COUNT_FAILED
+
+### 통합 (E2E)
+- `@SpringBootTest`로 mock OpenAiClient 빈 주입
+- 정상 흐름: 시니어 PUT (사진 + status=TAKEN, 동일 시각 schedule 2개, Vision mock=2) → ai_status=COUNT_MATCH 응답 검증
+- mismatch 케이스: Vision mock=1 (실제 2 예정) → ai_status=COUNT_MISMATCH
+- Vision 실패: mock throws → ai_status=COUNT_FAILED
+
+## 8) 오픈 질문
+| # | 질문 | 선택지 | 담당/기한 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)